apps/documents/serializers.py

```diff
@@ -106,3 +107,24 @@ def get_presigned_s3(self, obj):
         log_metric('transmission.info', tags={'method': 'documents.generate_presigned_url', 'module': __name__})
 
         return url
+
+    def get_presigned_s3_thumbnail(self, obj):
+        if obj.upload_status != UploadStatus.COMPLETE:
+            return super().get_presigned_s3(obj)
+
+        thumbnail_key = obj.s3_key[:-4] + '-t.jpg'
```

Comment:
I'd feel better about this if you split the string on the last '.' in the s3_key and took the left hand result. The current approach won't work if a file extension is more than 3 characters - i.e. 'james.jpeg'

Reply:
The reason I did it this way is that they are all saved as a .jpg, not a .jpeg in the s3 bucket through the thumbnail generator

Reply:
s3_key is a property that includes filename, which is a property that includes file_type, which is an enum that contains 3 extension types: pdf, jpeg, and png. Right now if a jpg is uploaded as a document, it will be renamed to jpeg (`uuid/uuid/uuid/uuid.jpeg`), and the thumbnail url that is generated from this line of code is:
`uuid/uuid/uuid/uuid.-t.jpg`